techniques/system/inventory/1.0/fusionAgent.st

```diff
@@ -55,8 +55,18 @@ bundle agent computeInventoryTime
 
 bundle agent doInventory
 {
+  vars:
+    resend_inventory::
+      "cant_send_inventory" string => "undefine", 
+                            classes => cancel_all_classes("cant_send_inventory");
+      
   classes:
-    "disable_inventory" expression => fileexists("${g.rudder_disable_inventory_file}");
+      "disable_inventory" expression => fileexists("${g.rudder_disable_inventory_file}");
```

Comment:
appart from this spacing change, all is great !

Reply:
this is properly indented contrarily to before

Reply:
ha yes, you are right